jobs/awslogs/templates/bin/pre-start

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "[general]"
+echo "state_file = /var/vcap/sys/log/awslogs/agent-state"
+
+for SYSTEM_LOG in /var/log/audit/audit.log /var/log/auth.log /var/log/dpkg.log /var/log/messages /var/log/syslog
```

Comment:
Hmm, I'm not really sure what that scenario you describe looks like -- I think I need more info. My objective here was just simplistic -- write out a single config file "awslogs.conf(.erb)" as needed.